src/utilities/TaskBuilder.cpp

```diff
@@ -29,8 +29,15 @@ TaskBuilder& TaskBuilder::SetCompletion(bool is_completed) {
     return *this;
 }
 
-TaskBuilder& TaskBuilder::SetLabel(const std::string &label) {
-    label_ = label;
+TaskBuilder& TaskBuilder::AddLabel(const std::string& label) {
+    labels_.push_back(label);
+    return *this;
+}
+
+TaskBuilder& TaskBuilder::RemoveLabel(const std::string& label) {
```

Comment:
A `Builder` pattern does not allow any "remove" logic. It's not a "Getter/Setter"
You create and fill it once to create an object. You can create multiple (same) objects, but you never change it to create another object.
If you "remove" label, you create another builder and don't set a label

@IlliaPlaksa @KharchenkoMaks @AntonApraksin @antonovchtd @MichaelKupriianov

Reply:
removed